rsmtool/writer.py

```diff
@@ -27,7 +38,7 @@ def __init__(self, experiment_id=None, context=None, wandb_run=None):
             The experiment name to be used in the output file names
```

Comment:
```suggestion
            The experiment name to be used in the output file names
            Defaults to ``None``
```
Although it's unusual that experiment_id is optional, is that correct?

Reply:
It's not experiment ID, it's the experiment name. I think it's always not `None` in the code but we need to allow `None` for tests.